habitat_sim/simulator.py

```diff
@@ -87,6 +87,16 @@ def seed(self, new_seed):
 
     def reset(self):
         self._sim.reset()
+        for agent_id in range(len(self.agents)):
```

Comment:
As mentioned in the issue https://github.com/facebookresearch/habitat-sim/issues/300 initial position of agents are not part of sim configuration right now. Therefore, best we can do just init agents position as it's done with `reconfigure` method:
```
        for i in range(len(self.agents)):
            self.initialize_agent(i)
```